### ISSUE TITLE
Fatal error on PHP 8.1.13 using path-mode intersection

### ISSUE THREAD

**Issue by jschaedl** [PHP-CS-Fixer/PHP-CS-Fixer#6718]
## Bug report

I encountered a problem running php-cs-fixer on PHP 8.1.13 using `path-mode` intersection. Here is the command I am running:

`php bin/php-cs-fixer fix --diff --dry-run -v --path-mode=intersection -- ./MyClass.php` which results in

```
Fatal error: Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, CallbackFilterIterator given in .../vendor/friendsofphp/php-cs-fixer/src/Console/Command/FixCommand.php:318
```

### Details

- PHP version: 8.1.13
- PHP CS Fixer version: 3.13.1

**Comment by jschaedl** [comment]
duplicate of #6715

**Comment by icanhazstring** [comment]
@jschaedl don't know if that is really a duplicate. These are two different errors.

**Comment by keradus** [comment]
Fix released as [v3.13.2](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v3.13.2)
